README.md

```diff
@@ -94,12 +94,7 @@ There is none. Every renderer should have complete Unicode support throughout,
 but every certificate template render method is built around English-grammar
 sentence interpolation. Consequently, if the target language of the certificate
 has very different sentence structure, you may find the easiest way to add
-support for your language is to fork an existing renderer. We suggest 
-```_generate_v3_dynamic_certificate```.
-
-If you think that your new renderer would be useful to the international Open
-edX community, please issue a pull request! We'd love to have 3_dynamic
-renderers for new languages.
+support for your language is to fork an existing renderer.
```

Comment:
Should we update this to say something like, "v4 should do everything you need, if not, pr to update it"?